docs/UserGuide.md

```diff
@@ -229,16 +231,7 @@ Example of usage:
 `set r 500`
 
 Expected Outcome:
```

Comment:
Perhaps instead of having example of usage + expected output, could you combine it into 1 sentence that reads something like: "the following diagram is an example of when `set r 500` command is used in a valid template"

Reply:
Okay, changed.